<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZegoAudioLiveClass</class>
 <widget class="QDialog" name="ZegoAudioLiveClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZegoAudioLive</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,0,2">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="m_AreaBeforePublish" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="m_lbUserID">
              <property name="styleSheet">
               <string notr="true">font-family: 微软雅黑;
font-size: 16px;</string>
              </property>
              <property name="text">
               <string>用户ID</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="m_edUserID">
              <property name="minimumSize">
               <size>
                <width>508</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>508</width>
                <height>32</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="m_lbUserName">
              <property name="styleSheet">
               <string notr="true">font-family: 微软雅黑;
font-size: 16px;</string>
              </property>
              <property name="text">
               <string>用户名</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="m_edUserName">
              <property name="minimumSize">
               <size>
                <width>508</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>508</width>
                <height>32</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="m_lbAppID">
            <property name="styleSheet">
             <string notr="true">font-family: 微软雅黑;
font-size: 16px;</string>
            </property>
            <property name="text">
             <string>App ID</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="m_edAppID">
            <property name="minimumSize">
             <size>
              <width>508</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>508</width>
              <height>32</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="placeholderText">
             <string>请输入AppID</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="m_lbAppSign">
            <property name="styleSheet">
             <string notr="true">font-family: 微软雅黑;
font-size: 16px;</string>
            </property>
            <property name="text">
             <string>App Sign</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QTextEdit" name="m_edAppSign">
            <property name="minimumSize">
             <size>
              <width>508</width>
              <height>158</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>508</width>
              <height>158</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="placeholderText">
             <string>请输入AppSign</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="m_cbAppVersion">
            <property name="minimumSize">
             <size>
              <width>128</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>128</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="m_checkTestEnv">
            <property name="minimumSize">
             <size>
              <width>127</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>127</width>
              <height>25</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">font-family: 微软雅黑;
font-size: 16px;</string>
            </property>
            <property name="text">
             <string>测试环境</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_checkManual">
            <property name="minimumSize">
             <size>
              <width>127</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>127</width>
              <height>25</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">font-family: 微软雅黑;
font-size: 16px;</string>
            </property>
            <property name="text">
             <string>手动开播</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_checkMute">
            <property name="minimumSize">
             <size>
              <width>128</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>128</width>
              <height>25</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">font-family: 微软雅黑;
font-size: 16px;</string>
            </property>
            <property name="text">
             <string>静音</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QListView" name="m_logList">
          <property name="minimumSize">
           <size>
            <width>588</width>
            <height>158</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>588</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 5px;
border: 1px solid gray;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="m_line">
     <property name="styleSheet">
      <string notr="true">margin-top: 13px;
margin-bottom: 13px;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="m_AreaAfterPublish" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>30</number>
        </property>
        <item>
         <widget class="QListView" name="m_memberList">
          <property name="minimumSize">
           <size>
            <width>380</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 5px;
border: 1px solid gray;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <widget class="QLineEdit" name="m_edSessionID">
            <property name="minimumSize">
             <size>
              <width>246</width>
              <height>27</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-family: 微软雅黑;
font-size: 16px;</string>
            </property>
            <property name="placeholderText">
             <string>请输入SessionID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_switchStartPublish">
            <property name="minimumSize">
             <size>
              <width>112</width>
              <height>34</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-family: 微软雅黑;
font-size: 16px;</string>
            </property>
            <property name="text">
             <string>进入</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ZegoMainDialog.qrc"/>
 </resources>
 <connections/>
</ui>
